File: processing-host/app/hms/AppStart.scala
package hms

object AppStart{
  import play.api._
  
  def main(args: Array[String]) {
    val env = Environment(new java.io.File("."), this.getClass.getClassLoader, Mode.Dev)
    val context = ApplicationLoader.createContext(env)
    val loader = ApplicationLoader(context)
    val app = loader.load(context)
    Play.start(app)  
  }
}

File: processing-host/app/hms/AppStart.java
package hms;
public class AppStart {
	public static void main(String[] args) {		
		play.Environment env = new play.Environment(new java.io.File("."), AppStart.class.getClassLoader(), play.Mode.DEV);
		play.ApplicationLoader.Context context = play.ApplicationLoader.create(env);
		play.ApplicationLoader loader = play.ApplicationLoader.apply(context);	
		play.Application app = loader.load(context);
		
		Thread t = new Thread(()->{play.api.Play.start(app.asScala());});
		t.start();
		System.console().readLine();
		play.api.Play.stop(app.asScala());	
		//AppStarter.start();
	}

}
